airbyte-integrations/connectors/source-okta/source_okta/schemas/users.json

```diff
@@ -1,3 +1,216 @@
 {
-  "$ref": "shared-user.json"
+  "type": ["object", "null"],
+  "additiona;Properties": true,
```

Comment:
typo (lots of them) - `additiona;Properties`

Reply:
Forgot to remove these `shared` schemas) Corrected.